society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -876,7 +915,33 @@
   <macro name="pages">
     <choose>
       <if type="article-journal">
-        <text variable="page" prefix=": "/>
+        <!-- Add support for individually paginated, electronic journal articles per https://sblhs2.com/2018/05/03/electronic-journals-with-individually-paginated-articles/ -->
+        <choose>
+          <if variable="number">
+            <text term="article-journal" form="verb-short" prefix=": "/>
+            <text variable="number" prefix=" " suffix=", "/>
+            <choose>
+              <if variable="locator">
+                <label variable="locator" form="short"/>
+                <text variable="locator"/>
+              </if>
+              <else>
+                <!-- Is there not a term for "no page"? See CMS17 §10.42 where the same abbreviation applies to "no place," "no publisher," and "no page." -->
+                <text value="n.p."/>
```

Comment:
this seems weird -- you'd potentially want to cite an electronic article without specifying a locator and shouldn't get "no pages" then, right?

Reply:
Yes, that's right—I just missed it when I was testing the update on my end. My apologies and thanks.